perun-base/src/main/java/cz/metacentrum/perun/core/api/RichGroup.java

```diff
@@ -53,7 +53,7 @@ public String toString() {
 		ret.append(this.getDescription());
 		ret.append("', voId='");
 		ret.append(this.getVoId());
-		ret.append("', groupAttributes='");
+		ret.append("', attributes='");
```

Comment:
AuditParser.java uses method `createRichGroup()` where it expects 'groupAttributes', that's where the tests fail.